Fix failing tests on master - adding more time to timeout

idanizi-panw opened the pull request:
at ci:
![image](https://user-images.githubusercontent.com/89729679/143733435-7e9f5656-2be8-4155-ac5a-e3ae347e7166.png)

locally
![image](https://user-images.githubusercontent.com/89729679/143733440-614b778b-5122-4b26-b21f-dde5ae188b24.png)

because of unstable test

asafshen commented on tests/containers/ReportContainer_spec.js:
```diff
@@ -310,7 +310,7 @@ describe('Report Container', () => {
     const toRender = <ReportContainer sections={prepareSections(testTemplate)} />;
     const reportContainer = mount(toRender);
 
-    await new Promise(resolve => setTimeout(resolve, 5001));
```
imo 10000 is more readable for humans

idanizi-panw replied:
will change

dorsha commented:
I don't understand the fix, please explain better. And why you choose this value for the timeout?

dorsha commented:
Also, what has changed that needed this change?

idanizi-panw commented:
> I don't understand the fix, please explain better. And why you choose this value for the timeout?

this test wasn't stable, needed more time for the report to render before checking expected.

> Also, what has changed that needed this change?

we added more time for `AutoSizer` to calculate the heights of elements and their borders so the text wont overflow

dorsha commented:
> > I don't understand the fix, please explain better. And why you choose this value for the timeout?
> 
> this test wasn't stable, needed more time for the report to render before checking expected.
> 
> > Also, what has changed that needed this change?
> 
> we added more time for `AutoSizer` to calculate the heights of elements and their borders so the text wont overflow

So the test was stable, what is the new time was added to the AutoSizer?

idanizi-panw commented:
> > > I don't understand the fix, please explain better. And why you choose this value for the timeout?
> > 
> > 
> > this test wasn't stable, needed more time for the report to render before checking expected.
> > > Also, what has changed that needed this change?
> > 
> > 
> > we added more time for `AutoSizer` to calculate the heights of elements and their borders so the text wont overflow
> 
> So the test was stable, what is the new time was added to the AutoSizer?

added 500ms instead of 100ms:
https://github.com/demisto/sane-reports/pull/195/files#diff-0e672ae5527a671a5aa65b657426c852c801bdc126cfb15da4da31528135593aR14
